lib/target_web/pow_mailer.ex

```diff
@@ -0,0 +1,22 @@
+defmodule TargetWeb.PowMailer do
+  @moduledoc false
+
+  use Pow.Phoenix.Mailer
+  use Bamboo.Mailer, otp_app: :target
+
+  import Bamboo.Email
+
+  def cast(%{user: user, subject: subject, text: text, html: html}) do
```

Comment:
If this is a struct it would be nice to pattern match it here

Reply:
Pattern match what exactly? @brunvez